### ISSUE TITLE
Question: Effect of Normalizaing Data wrt L1/L2 norm (sklearn.preprocessing.Normalizer() )

### ISSUE THREAD

**Issue by ArshChhabra** [mikeizbicki/cmc-csci145-math166#210]
I was not super sure about normalizing the data wrt L1 and L2 norm (normalize = sklearn.preprocessing.Normalizer(norm='l1'))

The output is x_i/norm(x)^2 for each i \in d. (with L1 norm being absolute sum and L2 norm being euclidean distance) so normalizing data is the same as scaling wrt norm(x)^2. Hence it should have the same effects as scaling and not affect statistical properties nor runtime. Ie we keep it if it helps and remove it if it doesnt. But the problem I see here is that each data point is being scaled by a different value and it is not the same scalar that multiplies every x \in \mathcal(X). Hence, do the regularization effects apply as given below?

<img width="1349" alt="Screenshot 2024-12-09 at 7 13 30 PM" src="https://github.com/user-attachments/assets/5a6bd7f6-b663-44dc-982f-b0650831198b">

My understanding is that the above image holds true and I just wanted to see if it was correct/incorrect. Thank you!!
